app/controllers/app.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'roda'
+require 'slim'
+
+module CodePraise
+  # Web App
+  class App < Roda
+    plugin :render, engine: 'slim', views: 'app/views'
+    plugin :assets, css: 'style.css', path: 'app/views/assets'
+    plugin :halt
+
+    route do |routing|
+      routing.assets # load CSS
+
+      # GET /
+      routing.root do
+        view 'home'
+      end
+
+      routing.on 'project' do
```

Comment:
these route blocks are going to get really big -- what should we do about rubocop's block length rules?